src/software/ai/hl/stp/stp.cpp

```diff
@@ -7,6 +7,7 @@
 #include <exception>
 #include <g3log/g3log.hpp>
 #include <random>
+#include <utility>
```

Comment:
Not sure why we added this include? If we need to include `shared_ptr`, we should use `#include <memory>` as `utility` contains more things that we probably don't need here.

Reply:
removed